tests/distributions/test_distributions.py

```diff
@@ -44,8 +44,12 @@ def test_infer_shapes(dist):
         pytest.xfail(reason="cannot statically compute shape")
     for idx in range(dist.get_num_test_data()):
         dist_params = dist.get_dist_params(idx)
-        arg_shapes = {k: v.shape if isinstance(v, torch.Tensor) else ()
-                      for k, v in dist_params.items()}
+        if 'SineSkewed' == dist.pyro_dist.__name__:
+            arg_shapes = {k: v.shape if isinstance(v, torch.Tensor) else v.batch_shape
+                          for k, v in dist_params.items()}
+        else:
+            arg_shapes = {k: v.shape if isinstance(v, torch.Tensor) else ()
+                          for k, v in dist_params.items()}
```

Comment:
It's probably best to revert this for now. At some future point we may add support for distribution arguments, but then we will do so more uniformly.

Reply:
That makes sense; I'll revert this.